joss_paper/paper.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 'alchemlyb: the simple alchemistry library'
+tags:
+  - Python
+  - alchemistry
+  - molecular dynamics
+  - free energy
+authors:
+  - name: Zhiyi Wu
+    orcid: 0000-0002-7615-7851
+    equal-contrib: true
+    affiliation: 1
+  - name: David L. Dotson
+    orcid: 0000-0001-5879-2942
+    equal-contrib: true
+    affiliation: "2, 3"
```

Comment:
@dotsdl I added ASU to your affiliations and marked Datryllic as the present one.

I think that this academically the right way to indicate that your work on alchemlyb started at ASU (with grant support) and then continued at Datryllic.

If you think that this is problematic in some form please say something.